src/ui/components/atoms/label-icon/styles.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+}
+
+.label {
```

Comment:
You are not using BEM, this code can affect other components and collide with these styles, this is very bad practice.